sources.cfg

```diff
@@ -2,5 +2,6 @@
 extends = http://plonesource.org/sources.cfg
 
 auto-checkout =
+    ftw.simplelayout
```

Comment:
What is the reason, you need to install `ftw.simplelayout` from source? What feature do you need, which has not been released yet? As I can see in https://github.com/4teamwork/ftw.simplelayout/blob/20904427ba7c5337dd1625fe6892c323a9a77b7d/docs/HISTORY.txt, these are the unreleases changes in `ftw.simplelayout`:

```
- Staging: remove auto generated children when creating a working copy. [jone]

- Staging: Add link to working copies in warning message. [mathias.leimgruber]

- Fix plone 4 build by pinning collective.z3cform.datagridfield to 1.4.0. [mathias.leimgruber]
```

Reply:
I forgot to put the changelog there. It is because of reverting commit `abc8`. The changelog is in review in another pull request, but not yet merged.
If you read the comment above jone told me to put sl into autocheckout ;)

Reply:
👍